pkg/common/config/types.go

```diff
@@ -101,6 +103,11 @@ type FeatureStatesConfigInfo struct {
 	Namespace string
 }
 
+// TopologyCategoryInfo contains metadata for the Zone and Region parameters under Labels section.
+type TopologyCategoryInfo struct {
+	Label string
```

Comment:
Can you check if `gcfg:` is required for this string?

Reply:
Good catch, added it.